bundle/edu.gemini.dataman.app/src/main/scala/edu/gemini/dataman/app/Dataman.scala

```diff
@@ -98,7 +113,8 @@ object Dataman {
       schedulePoll(pollArchive.thisWeek, config.thisWeekPeriod)
 
       schedule(progSync, progSyncDelay, config.allPeriod.time)
-      schedule(new RetryFailedRunnable(User, retryMinDelay, exec, odb), retryMinDelay.plusMillis(1), retryPeriod)
+      schedule(new ObsRefreshRunnable(odb, User, pollService), obsRefreshDelay, config.obsRefreshPeriod.time)
```

Comment:
I don't recall, if a task fails with an exception, would the scheduler retry it in the next period?